fileio/ft_read_header.m

```diff
@@ -15,6 +15,8 @@
 %   'coordsys'       = string, 'head' or 'dewar' (default = 'head')
 %   'chantype'       = string or cell of strings, channel types to be read (NeuroOmega, BlackRock).
 %   'headerformat'   = name of a MATLAB function that takes the filename as input (default is automatic)
+%   'password'       = password structure for encrypted data set (such as mayo_mef30 and mayo_mef21)
+%   'sortchannel'    = sort channel order according to either alphabet or number (default is alphabet)
```

Comment:
the `sortchannel` option allows the output of ft_read_header and ft_read_data to become inconsistent if the user were to use it with one, and not the other. Sorting channels is trivial to do outside of ft_read_header and ft_read_data and therefore I don't see the reason to implement it as part of the functions. If anywhere, it would make more sense to implement it in ft_preprocessing.

Reply:
That's a good point.  A mechanism to make consistent between `ft_read_header` and `ft_read_data` has already built in the code.  Sorting channel according to channel number is of practical importance and is not that straightforward, as the user needs to go deep into the fields of header.  However, we'll discuss whether to sort the channels in `ft_read_header` and `ft_read_data`.